lib/Component/VCard.php

```diff
@@ -411,6 +411,24 @@ function preferred($propertyName) {
     }
 
     /**
+     * Returns a specific type of multi fields (ADR, TEL, or EMAIL).
+     *
+     * @param string $propertyName
+     * @param string $type
+     *
+     * @return VObject\Property|null
+     */
+    function getForType($propertyName, $type) {
+        foreach ($this->select($propertyName) as $field) {
+            if (isset($field['TYPE']) && $field['TYPE']->getValue() === $type) {
```

Comment:
What happens in a case when there's a property with multiple TYPE parameters, or a multi-value TYPE parameter? I think Parameter objects have a `->has` function that should be used instead.